js/apps/admin-ui/src/i18n.ts

```diff
@@ -8,10 +8,78 @@ import { getAuthorizationHeaders } from "./utils/getAuthorizationHeaders";
 import { addTrailingSlash } from "./util";
 
 export const DEFAULT_LOCALE = "en";
+const OVERRIDES_NS = "overrides";
+
+/**
+ * A ResourceResult can actually be of ANY type.
+ */
+type ResourceResult = any;
+
+/**
+ * Helper type to define internal operations of ResourceStore, which we have to access.
+ */
+interface ResourceStoreInternal extends ResourceStore {
+  getResource(
+    lng: string,
+    ns: string,
+    key?: string | string[],
+    originalOptions?: Pick<InitOptions, "keySeparator" | "ignoreJSONStructure">
+  ): ResourceResult;
+}
 
 export async function initI18n(adminClient: KeycloakAdminClient) {
   const options = await initOptions(adminClient);
   await init(options);
+
+  addInterceptorForGetResource();
+}
+
+/**
+ * Intercept the "getResource" function of the resource store, in order to apply the "overrides" from realm localization.
+ *
+ * An alternative would be to use a postProcessor, but to make that work, almost all the default processing logic
+ * would need to be re-implemented here.
+ * Several functionality cannot be easily supported with a postProcessor:
+ * <ul>
+ *     <li>The key might or might not contain the namespace, thus key parsing would need to be re-implemented.</li>
+ *     <li>In case of plurals, the key will not contain the "_one" or "_other" suffix - would need to be re-implemented</li>
+ *     <li>For the "overrides", interpolation (replacing placeholders) would need to be applied.</li>
+ *     <li>Probably more ...</li>
+ * </ul>
+ */
+function addInterceptorForGetResource() {
```

Comment:
Perhaps we should be implementing this as a custom backend instead? See the documentation on [Backend Fallback](https://www.i18next.com/how-to/backend-fallback).

Reply:
I also thought of implementing such a fallback, but from my POV it is not possible (at least without a bigger refactoring of i18next):

As far as I understand, the Backend fallback works a bit different as one might expect. It just tries a different (fallback) locale/namespace URL, when the first one cannot be loaded at all (e.g. returns status 404). Hence, it does not work when some message keys are translated and some are not.